dashboards/dashboards.libsonnet

```diff
@@ -4,4 +4,5 @@
 (import 'resources.libsonnet') +
 (import 'statefulset.libsonnet') +
 (import 'use.libsonnet') +
-(import 'defaults.libsonnet')
+(import 'defaults.libsonnet') +
+(import 'windows.libsonnet')
```

Comment:
having windows compute still seems somewhat esoteric at this point so I don't think this should be the default, if users want this they should just import it themselves.

Reply:
done.